pkg/workflows/wasm/pb/wasm.go

```diff
@@ -75,7 +75,7 @@ func WorkflowSpecToProto(spec *sdk.WorkflowSpec) (*WorkflowSpec, error) {
 		if err != nil {
 			return nil, fmt.Errorf("error translating step definition to proto: %w", err)
 		}
-		ws.Targets = append(ws.Consensus, tt)
```

Comment:
Whoops :) Thanks!

Reply:
Can you also extend the corresponding tests so that this is regression tested?

Reply:
Extended tests as suggested